recommendation/templates/recommendation/recommendation_rating.html

```diff
@@ -0,0 +1,189 @@
+
+{% load static %}
+<link href="{% static 'css/bootstrap.min.css' %}" rel="stylesheet">
+<link rel="stylesheet" type="text/css" href="{% static 'recommendation/css/recommendation.css' %}">
+<!-- Load bootstrap css, jquery, bootstrap js -->
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```

Comment:
Preferably, the fix for including the common jquery, bootstrap js, and bootstrap css files in the dashboard app would be able to take care of these inclusions for you (if we extend dashboard/index.html). That way we can make sure that the common styles aren't overloaded by a particular application and the website is coherent.

Reply:
I do not have the convenience of testing this page on my local, so please ignore this if you're opening this in a new tab (but please do comment if that's the case). Following up on the above, reading the code base suggests that we have this is a separate page with the dashboard styles copied. If this works as it's supposed to we can ignore this comment and merge this for this sprint, but maybe we can revisit this in the next sprint (if required).

Reply:
Yes, it is a separate page that is not linked to the dashboard and just copies its style to keep it looking uniform to the theme of the site.